enterprise/validators.py

```diff
@@ -48,3 +50,22 @@ def validate_image_size(image):
     if config and not image.size <= valid_max_image_size_in_bytes:
         raise ValidationError(
             _("The logo image file size must be less than or equal to %s KB.") % config.valid_max_image_size)
+
+
+def validate_content_filter_fields(content_filter):
+    """
+    Validate particular fields (if present) passed in through content_filter are certain types.
+    """
+    for key in CONTENT_FILTER_FIELD_TYPES:
+        if key in content_filter.keys():
+            if not isinstance(content_filter[key], CONTENT_FILTER_FIELD_TYPES[key]['type']):
```

Comment:
minor point: could we store `CONTENT_FILTER_FIELD_TYPES[key]` as a constant in this scope (or if block) given it's used a few times ?

Reply:
oh you mean to just to shorten it? like `cf_key = CONTENT_FILTER_FIELD_TYPES[key]` ?

Reply:
I went with shortening the import. The extra assignment isn't really costly, but might as well just make the whole constant as we use it shorter.